### PR TITLE
Pull `forgerock-parent-2.0.10` into `master` and upgrade PGPVerify

### LINKED ISSUE #7: Verifying signature of locally installed tools.jar fails
## Summary

The parent POM specifies the locally installed `tools.jar` should be used (which is a good thing). However verifying the PGP signature fails unless you have the exact version which is specified in the hosted `trustedkeys.properties` file.

It is impractical to specify a signature for all possible `tools.jar` JAR's. Even if we would do so there is an other problem we would bump into; The same versions of `tools.jar` can have different signatures. For example I use CentOS 7.4 which uses an OpenJDK version build by the CentOS project. However the same version by Ubuntu will probably have a different signature because for example the `MANIFEST.MF` file contains the version of the JDK used to create the `tools.jar` JAR.

Since `tools.jar` is part of the JDK installation I think verifying integrity of  `tools.jar` is out of scope for the WrenSecurity project; It's something the distribution takes care of by using signed packages. If `tools.jar` is compromised the compiler or Maven itself might be compromised as well.

## Steps
1. checkout Wren:DS (or any Wren project that inherits from `wrensec-parent`) 
2. Verify you use a different JDK version then is specified for `tools.jar` in `trustedkeys.properties`.
3. run `mvn clean install` in the root of the checked-out project.

## Expected
Project builds without issue.

## Current

Build fails with the follwoing error:
```
[ERROR] Failed to execute goal com.github.s4u.plugins:pgpverify-maven-plugin:1.2.0-SNAPSHOT:check (default) on project opendj-config: No signature for com.sun:tools:jar:1.8.0_144 -> [Help 1]
```

## Workarounds
- `mvn -Dignore-artifact-sigs clean install` works, but that completely skips PGP verification of artifacts (less secure).

### PR DESCRIPTION
**Pull request by Kortanul**
- Pulls in ForgeRock's 2.0.10 POM file into the `master` branch (I originally created just a sustaining branch for it).
- Upgrades PGPVerify to 1.2.0-wren1 so we can stop having issues with provided & system JARs (closes #7).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!--
  DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 
- Copyright 2011-2015 ForgeRock AS.
+ Copyright 2011-2016 ForgeRock AS.
```
Comment (siepkes) [inline pom.xml:5]: Is updating FR copyright entries something we should be doing? Shouldn't we just leave them alone?

**Review by siepkes — CHANGES_REQUESTED** [review]
Only have a question about the license header, otherwise looks good to me!

File: pom.xml (hunk 1 above)
Comment (Kortanul) [reply pom.xml:5]: @siepkes ForgeRock actually updated it. This was constructed from their sources, to form 2.0.10.

**Comment by Kortanul** [comment]
Back to you @siepkes

**Review by siepkes — APPROVED** [review]
Ah yes, of course, silly me! LGTM in that case!

**Comment by Kortanul** [comment]
Thanks!

**Pull request merged**